Bottom: 67024f81d01ffab43c8ad6a4b5599a6f79e33feb
Top:    81b5fe9c821b656d996dc3f142a6456b9aaf6055
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-03-12 01:25:59 +0000

Make --enable-dbus default


---

diff --git a/configure.ac b/configure.ac
index 26b20cb..c6f27e5 100644
--- a/configure.ac
+++ b/configure.ac
@@ -353,7 +353,13 @@ fi
 AC_MSG_CHECKING([for whether to use DBUS])
 AC_ARG_ENABLE([dbus],
 [  --enable-dbus           Enable DBUS IPC],
-[],[enable_dbus=no])
+[],[
+	case " $with_gui " in
+		*\ gtk\ *)     enable_dbus=yes ;;
+		*\ lesstif\ *) enable_dbus=yes ;;
+		* )            enable_dbus=no ;;
+	esac
+])
 
 AC_MSG_RESULT([$enable_dbus])
 AM_CONDITIONAL(WITH_DBUS, test x$enable_dbus = xyes)
